thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/merge/AnomalyMergeConfig.java

```diff
@@ -19,6 +22,17 @@
   private AnomalyMergeStrategy mergeStrategy = AnomalyMergeStrategy.FUNCTION;
   private long sequentialAllowedGap = 30_000; // 30 seconds
   private long maxMergeDurationLength = 12 * 60 * 60 * 1000; // 12 hours
+  private List<String> mergeablePropertyKeys = new ArrayList<>();
+
+  public AnomalyMergeConfig() {
+  }
+
+  public AnomalyMergeConfig(AnomalyMergeConfig other) {
```

Comment:
somehow it looks weird to have this kind of constructor. It might be better to define:
mergeConifg(mergeStrategey, gap, length, keys) and use other.* as input when you need to construct a new config with an existing config.